Fixes missing wire on boxstation

YPOQ opened the pull request:
Was really annoying since this meant the R&D/departures APC wasn't being charged.

Before:
![before](https://user-images.githubusercontent.com/30683121/36718583-8a05c29e-1b5f-11e8-8ee3-0ffea61b6b2f.PNG)
After:
![after](https://user-images.githubusercontent.com/30683121/36718590-94fe6336-1b5f-11e8-99da-c07317bc8168.PNG)

Robustin commented:
Ah god I fixed this in my Box update but I thought I had introduced the problem too.

Cyberboss reviewed (approved):
Signed off by: @Cyberboss

YPOQ committed:
Fixes missing wire on boxstation - (#36011)
Was really annoying since this meant the R&D/departures APC wasn't being charged.

Before:
![before](https://user-images.githubusercontent.com/30683121/36718583-8a05c29e-1b5f-11e8-8ee3-0ffea61b6b2f.PNG)
After:
![after](https://user-images.githubusercontent.com/30683121/36718590-94fe6336-1b5f-11e8-99da-c07317bc8168.PNG)

284719103c2b04069ce407fcbd9ad072d15dea4e
